scripts/src/github.py

```diff
@@ -32,41 +32,47 @@ def getCommitId(branch: str):
     repo = "SEPTIC"
     endpoint = f"/repos/{owner}/{repo}/branches/{branch}"
     response = sendRequestGithub(endpoint)
-    return response["commit"]["sha"][0:7]
+    return response["commit"]["sha"]
 
 
-def getCalcFile(branch: str):
+def getCalcFile(ref: str):
```

Comment:
`ref` or `commit` as mentioned above? Same for `getFile()`, `getDir()`, `getObjectFiles()`,